packages/traceloop-sdk/examples/dataset_attachments_example.py

```
@@ -0,0 +1,357 @@
+                    filename="analysis_config.json",
+                    content_type="application/json",
+                    file_type=FileCellType.FILE,
+                    metadata={"version": "1.0", "algorithm": "regression"},
+                ),
+            },
+        ],
+    )
+
+    dataset = datasets.create(dataset_request)
+    print(f"Created dataset: {dataset.slug}")
+    print(f"Attachments uploaded from memory")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)